sdk/core/azure-core-jackson-tests/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure Core Jackson Tests plugin library for Java
+
+Azure Core Jackson Tests is a test package that tests `azure- core` library against multiple versions of Jackson libraries that users might have.
```

Comment:
why the different spacing with 
`azure- core`
`azure - core - test`
`azure-core-jackson-tests`

Reply:
good catch! `azure- core` is a typo. `azure-core` and `azure-core-jackson-tests` are package names. 

`java- core - test` (another typo!) is a test pipeline and this is how its name looks. I'd give a link to it, but it's an internal pipeline 

![image](https://user-images.githubusercontent.com/2347409/128270931-b2625876-1fef-4c7a-a134-2d7809ae7502.png)